app/assets/stylesheets/common.scss

```diff
@@ -2092,6 +2092,7 @@ textarea {
   color: #222;
   background-color: #fff;
   border: 1px solid $grey;
+  border-radius: 3px;
```

Comment:
This should use the `$border-radius` variable

Reply:
I was initially reluctant to use the variable, because it is only used for the navigation bar buttons in the CSS. All the other items that make use of `border-radius` instead have the 3px hardcoded. I thought that `$border-radius` is only used for navigation buttons and if their border-radius is changed in the future, it might look weird if suddenly all sorts of things (e.g. the browse tag list) also have a larger `border-radius`. Should I change it anyway?

Reply:
I think this could be debated either way, but it's not super important so let's leave it as you've proposed.